src/app/components/landing-page/landing-page.component.ts

```diff
@@ -144,12 +147,14 @@ export class LandingPageComponent implements OnInit, AfterViewInit, OnDestroy {
       this.refreshSubscription = this.arlasIamService.tokenRefreshed$.subscribe({
         next: (loginData) => {
           if (!!loginData) {
+            this.builderForOrgs = loginData.user.roles.filter(r => r.name === 'role/arlas/builder').map(r => r.organisation.name);
```

Comment:
Is it possible to put it in a global const, just in case one day this role changes

Reply:
No longer relevant